content/productivity/distribute-silver/plugin.js

```diff
@@ -63,14 +64,43 @@ class Plugin {
       }
     }
 
+    // asteroid level
+
+    let levelAsteroidLabel = document.createElement('label');
+    levelAsteroidLabel.innerText = 'Max. Lvl asteroid from:';
+    levelAsteroidLabel.style.display = 'block';
+
+    let levelAsteroid = document.createElement('select');
+    levelAsteroid.style.background = 'rgb(8,8,8)';
+    levelAsteroid.style.width = '100%';
+    levelAsteroid.style.marginTop = '10px';
+    levelAsteroid.style.marginBottom = '10px';
+    [0, 1, 2, 3, 4, 5, 6, 7].forEach(lvl => {
```

Comment:
Should we support up to level 9 since that is max this round? Also, could you update the other list? Maybe define as a const at the top of file?

Reply:
I did a quick fix for this one. I will refactor it later. I'm working on a v2 with more cleaning